Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.8-slim as dev
```

Comment:
Esse apelido `as dev` me parece que não é utilizado, né? Se não for, acho que remover pode ser positivo, pois não cria a expectativa de que vai ter uma [_build_ com múltiplas fases](https://docs.docker.com/develop/develop-images/multistage-build/).